src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/arrival/Arrival.kt

```diff
@@ -36,7 +36,10 @@ data class Arrival(
   val fromLocationType: LocationType,
 
   @Schema(description = "Is the arrival in custody according to NOMIS", example = "true")
-  val isCurrentPrisoner: Boolean = false
+  val isCurrentPrisoner: Boolean = false,
+
+  @Schema(description = "Gender of the arrival according to Book a Secure Move", example = "MALE")
```

Comment:
Might be better to add a comment here that we can only map across male and female, if BASM returns trans, then we pass back null as we need to explicitly capture legal gender information.